server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimGroupMembershipManager.java

```diff
@@ -502,4 +529,24 @@ public ScimGroupMember mapRow(ResultSet rs, int rowNum) throws SQLException {
         }
 
     }
+
+    private static final class ScimGroupRowMapper implements RowMapper<ScimGroup> {
```

Comment:
this seems to be a duplicate class. Each time we change the scim group we would be bound to change it in two places.
We will review and optimize if needed.